Frontend/ReactAndGlobalCSS/README.md

```diff
@@ -0,0 +1,28 @@
+**Title:** Working with Global CSS in ReactJS and ReactNative
+**Creator:** Tenji Tembo
+
+### Summary
+
+ReactJS is one of the most commonly used Front end frameworks in the modern web. It's component-first philosophy drives the direction and consistency of the platform, include areas such as CSS. So much so that CSS is "componentized" into ReactJS files and left unable to take advantage of more global elements found in more custom css libraries, such as heavily modified Bootstrap Libraries. In this talk, I'll talk about the available cohesion of using globally available CSS libraries, while still giving overall control to the component referenced via state management.
```

Comment:
General comment, this section is a little too verbose. We can probably halve it by not going too deep into how React creates components and the bootstrap details